src/classes/RD2_EnablementService.cls

```diff
@@ -104,4 +104,40 @@ public with sharing class RD2_EnablementService {
         return 
             UTIL_CustomSettingsFacade.getRecurringDonationsSettings().isRecurringDonations2Enabled__c == true;
     }
+
+    /**
+     * @description Returns true if RecurringType__c 'Fixed' picklist option available
+     */
+    @TestVisible
```

Comment:
`@TestVisible` isn't needed if this is public.

Reply:
That's needed in order to access the private setter.

Reply:
Good point.